kong/requirements.txt

```diff
@@ -0,0 +1 @@
+# integration pip requirements
```

Comment:
We're not putting requests in here, so  I don't think we should here either, we should rely on the agents request version and the agent simplejson version.